charts/harvester-cloud-provider/harvester-cloud-provider/100.0.0+up0.1.4/templates/rbac.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: {{ include "harvester-cloud-provider.name" . }}
+  namespace: {{ .Release.Namespace }}
+  labels:
+  {{- include "harvester-cloud-provider.labels" . | nindent 4 }}
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: {{ include "harvester-cloud-provider.name" . }}
+rules:
+  - apiGroups: [ "" ]
+    resources: [ "services", "nodes", "events" ]
+    verbs: [ "get", "watch", "list", "update", "create", "patch" ]
+  - apiGroups: [ "" ]
+    resources: [ "nodes/status", "services/status" ]
+    verbs: [ "update", "patch" ]
+  - apiGroups: [ "coordination.k8s.io" ]
+    resources: [ "leases" ]
+    verbs: [ "get", "update", "create" ]
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: {{ include "harvester-cloud-provider.name" . }}
+  labels:
+  {{- include "harvester-cloud-provider.labels" . | nindent 4 }}
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: {{ include "harvester-cloud-provider.name" . }}
+subjects:
+  - kind: ServiceAccount
+    name: {{ include "harvester-cloud-provider.name" . }}
+    namespace: {{ .Release.Namespace }}
```

Comment:
There aren't any user roles here that aggregate to the rancher roles (project-owner/project-member), are those needed ? Ex: https://github.com/rancher/charts/blob/dev-v2.6/charts/rancher-logging/rancher-logging/100.0.1%2Bup3.15.0/templates/userroles.yaml

Reply:
No.